onboard/cv/artag_detector.cpp

```diff
@@ -34,6 +34,8 @@ TagDetector::TagDetector() {  //initializes detector object with pre-generated d
     // initialize other special parameters that we need to properly detect the URC (Alvar) tags
     alvarParams = new cv::aruco::DetectorParameters();
     alvarParams->markerBorderBits = 2;
+    alvarParams->doCornerRefinement = false; //change 1
```

Comment:
**Nit** What does `change 1` mean here? If it's a setting that may need to be changed by others during testing, make sure the comment is more clear.

Reply:
Not sure. Probably an old comment